apps/next/app/dashboard/page.tsx

```diff
@@ -45,6 +46,53 @@ async function findCreateUserWithClerkId({
   return newUser;
 }
 
+const ensureTeamForClerkOrg = async ({
+  clerkOrgId,
+  teamName,
+  userId,
+}: {
+  clerkOrgId: string;
+  teamName: string;
```

Comment:
I don't think we want to have a team name argument here--it should be whatever the clerk organization's name is

Reply:
Done.